nbs/examples/Multivariate_with_TSMixer.ipynb

```diff
@@ -16,11 +16,11 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.                    val_check_steps=100,
</pre><p>I believe it is missing the <code>futr_exog_list</code> parameter to add the exogenous covariates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Ah, you're right, stupid from me, I thought they were picked up automatically from the dataframe but that's not the case of course if they are not specified in <code>futr_exog_list.</code></p>

Reply:
Added `futr_exog_list `to TSMixerx in the example, and reran the experiments